db/system_keyspace_sstables_registry.hh

```diff
@@ -15,23 +15,23 @@ class system_keyspace_sstables_registry : public sstables::sstables_registry {
 public:
     system_keyspace_sstables_registry(system_keyspace& keyspace) : _keyspace(keyspace.shared_from_this()) {}
 
-    virtual seastar::future<> create_entry(sstring location, sstring status, sstables::sstable_state state, sstables::entry_descriptor desc) override {
+    virtual seastar::future<> create_entry(utils::UUID location, sstring status, sstables::sstable_state state, sstables::entry_descriptor desc) override {
```

Comment:
Here it should be table_id, which can't be mixed with another uuid. Perhaps in system_keyspace too - it's declared in schema_fwd.hh, so in a fairly low-level module.